modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -908,6 +895,187 @@ The number of transform processors in a specific state (running, inactive, error
 
 ---
 
+=== Cloud storage metrics
+
+This section provides reference descriptions of the metrics available when Redpanda's xref:manage:tiered-storage.adoc[] is enabled.
```

Comment:
I noticed that the other sections don't really have descriptions following the heading... It's probably nice if we did that for all of them, but I'm also looking at the Prerequisites below and it has the same info as this paragraph. If I had to pick I'd just go with the prerequisites -- maybe as an admonition or an H4

Reply:
Adjusted